doc/source/dglc.rst

```diff
@@ -0,0 +1,121 @@
+.. _dglc:
+
+Data Land-Ice (DGLC)
+=================
+
+DGLC is normally used as a substitute for CESM/CISM running in NOEVOLVE mode.
```

Comment:
I wonder if this should be written in a more model agnostic way?  Will all users of dglc be familiar with cesm and/or cism?

Reply:
@jedwards4b - no one other than CESM and NorESM uses  a land-ice component right now - and the whole point of the dglc was to remove the need to run with CISM running in noevolve mode. @billsacks - can you think of another way to state this? I can't.